domains/measurements/Measurements.API/Tests/Fixtures/TestServerFixture.cs

```diff
@@ -0,0 +1,229 @@
+#region Copyright notice and license
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/grpc/test-services/sample/Tests/Server/IntegrationTests/Helpers/GrpcTestFixture.cs
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using EnergyOrigin.TokenValidation.Utilities;
+using EnergyOrigin.TokenValidation.Values;
+using Grpc.Net.Client;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration.Memory;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Tests.Fixtures.TestServerHelpers;
+using Xunit.Abstractions;
+
+namespace Tests.Fixtures
+{
+    public delegate void LogMessage(LogLevel logLevel, string categoryName, EventId eventId, string message, Exception? exception);
+
+    public class TestServerFixture<TStartup> : IDisposable where TStartup : class
+    {
+        public byte[] PrivateKey { get; set; } = RsaKeyGenerator.GenerateTestKey();
+
+        private TestServer? _server;
```

Comment:
it seems that you are using _ on all the private fields. It is not a naming convention we are using in EO. Why differ?

Reply:
It's been copy pasted from the wallet which has it from somewhere else